Cargo.lock

```diff
@@ -240,6 +240,12 @@ version = "1.3.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "bef38d45163c2f1dde094a7dfd33ccf595c92905c8f8f4fdc18d06fb1037718a"
 
+[[package]]
+name = "bitflags"
```

Comment:
Seems like `bitflags` is now included two times

Reply:
I will try to investigate that issue.